src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedParameterSymbol.cs

```diff
@@ -178,6 +178,11 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
             {
                 AddSynthesizedAttribute(ref attributes, moduleBuilder.SynthesizeIsReadOnlyAttribute(this));
             }
+
+            if (this.IsParams)
+            {
+                AddSynthesizedAttribute(ref attributes, compilation.TrySynthesizeAttribute(WellKnownMember.System_ParamArrayAttribute__ctor));
```

Comment:
>System_ParamArrayAttribute__ctor [](http://example.com/codeflow?start=107&length=32)

Is this an optional attribute or is this code going to fail if the attribute is bad or missing? Consider adding a test attempting to emit code with `paramarray` and the attribute missing.